p2p/peerexchange/discovery.go

```diff
@@ -49,7 +49,7 @@ type Discovery struct {
 }
 
 // New creates a Discovery instance.
-func New(logger log.Log, h host.Host, config Config) (*Discovery, error) {
+func New(logger log.Log, h host.Host, config Config) (*Discovery, bool, error) {
```

Comment:
this should not be a part of New signature

Reply:
rm'ed